core-java/src/test/java/com/baeldung/nulls/UsingObjectsUnitTest.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.nulls;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class UsingObjectsUnitTest {
+
+    private UsingObjects classUnderTest;
+
+    @BeforeEach
+    public void setup() {
+        classUnderTest = new UsingObjects();
+    }
+
+    @Test
+    public void whenArgIsNull_thenThrowException() {
+
+        assertThrows(Exception.class, () -> classUnderTest.accept(null));
```

Comment:
Objects.requireNonNull throws NullPointerException, according to the Javadoc.
```suggestion
        assertThrows(NullPointerException.class, () -> classUnderTest.accept(null));
```